0-ad-history-mod-package/simulation/templates/units/han/cavalry_archer_b.xml

```diff
@@ -8,6 +8,12 @@
     <Requirements>
       <Techs datatype="tokens">phase_town</Techs>
     </Requirements>
+    <History>
+        Horses were vital for the Han military in combat against the nomadic horsemen of the North. Because combat-capable horses were, for this reason, a precious good, the possession and provision of horses for individuals were strictly regulated. There was, for example, a law in place that allowed families to exempt three men from military service if they could provide a strong, healthy horse instead.
```

Comment:
This first paragraph seems fragmented. It jumps from the Han needing horses to the Han regulating them to the Han having a rule to let them get warhorses, and there is no through line.

You could rewrite it, but personally I think most is cut and the sentence from the too short paragraph below is added, it makes a pretty strong paragraph.

```suggestion
       War horses were vital in the Han military's fight against the nomadic horsemen of the North. However, indigenous Chinese breeds were comparably small and too weak for use in battle. This forced the Han to resort to other means of obtaining horses for their cavalry.
```